### ISSUE TITLE
Tailscale 1.25.91 service does not start on Windows 10

### ISSUE THREAD

**Issue by DentonGentry** [tailscale/tailscale#4744]
### What is the issue?

After installing Tailscale 1.25.91 on a Windows 10 system, the service does not start and the Taskbar icon says to restart the tailscale service. Starting tailscaled from a cmd.exe does run without errors.

The error appears to be `ipnserver: rename .\..tmp1113218995 : The system cannot find the path specified` though I don't know what `tmp1113218995` is.

```
2022-05-26 20:54:12.9829467 -0700 PDT: Program starting: v1.25.91-tfc5839864-g75e9154c1: []string{"C:\\Program Files (x86)\\Tailscale IPN\\tailscaled.exe", "/subproc", "2646014839487dfcdbf94544078db3865f63fa7a9ce8cacd2b799d3b7c4c5570"}
2022-05-26 20:54:12.9835017 -0700 PDT: subproc mode: logid=2646014839487dfcdbf94544078db3865f63fa7a9ce8cacd2b799d3b7c4c5570
2022-05-26 20:54:13.0203334 -0700 PDT: tailscaled: getting engine... (try 1)
2022-05-26 20:54:13.0886801 -0700 PDT: ipnserver: rename .\..tmp1113218995 : The system cannot find the path specified.
2022-05-26 20:54:13.1109256 -0700 PDT: subprocess exited: exit status 1
2022-05-26 20:54:13.1109256 -0700 PDT: BabysitProc: backoff: 30953 msec
2022-05-26 20:54:44.0784821 -0700 PDT: exec: "C:\\Program Files (x86)\\Tailscale IPN\\tailscaled.exe" [/subproc 2646014839487dfcdbf94544078db3865f63fa7a9ce8cacd2b799d3b7c4c5570]
2022-05-26 20:54:44.5457764 -0700 PDT: Program starting: v1.25.91-tfc5839864-g75e9154c1: []string{"C:\\Program Files (x86)\\Tailscale IPN\\tailscaled.exe", "/subproc", "2646014839487dfcdbf94544078db3865f63fa7a9ce8cacd2b799d3b7c4c5570"}
2022-05-26 20:54:44.5457764 -0700 PDT: subproc mode: logid=2646014839487dfcdbf94544078db3865f63fa7a9ce8cacd2b799d3b7c4c5570
2022-05-26 20:54:44.623996 -0700 PDT: tailscaled: getting engine... (try 1)
2022-05-26 20:54:44.6512458 -0700 PDT: ipnserver: rename .\..tmp1738592070 : The system cannot find the path specified.
2022-05-26 20:54:44.6760241 -0700 PDT: subprocess exited: exit status 1
2022-05-26 20:54:44.6760241 -0700 PDT: BabysitProc: backoff: 15682 msec
2022-05-26 20:55:00.3728352 -0700 PDT: exec: "C:\\Program Files (x86)\\Tailscale IPN\\tailscaled.exe" [/subproc 2646014839487dfcdbf94544078db3865f63fa7a9ce8cacd2b799d3b7c4c5570]
2022-05-26 20:55:00.703769 -0700 PDT: Program starting: v1.25.91-tfc5839864-g75e9154c1: []string{"C:\\Program Files (x86)\\Tailscale IPN\\tailscaled.exe", "/subproc", "2646014839487dfcdbf94544078db3865f63fa7a9ce8cacd2b799d3b7c4c5570"}
2022-05-26 20:55:00.7043813 -0700 PDT: subproc mode: logid=2646014839487dfcdbf94544078db3865f63fa7a9ce8cacd2b799d3b7c4c5570
2022-05-26 20:55:00.7463042 -0700 PDT: tailscaled: getting engine... (try 1)
2022-05-26 20:55:00.7800074 -0700 PDT: ipnserver: rename .\..tmp1506801549 : The system cannot find the path specified.
2022-05-26 20:55:00.7960105 -0700 PDT: subprocess exited: exit status 1
2022-05-26 20:55:00.79658 -0700 PDT: BabysitProc: backoff: 24203 msec
2022-05-26 20:55:25.0081479 -0700 PDT: exec: "C:\\Program Files (x86)\\Tailscale IPN\\tailscaled.exe" [/subproc 2646014839487dfcdbf94544078db3865f63fa7a9ce8cacd2b799d3b7c4c5570]
2022-05-26 20:55:25.6185322 -0700 PDT: Program starting: v1.25.91-tfc5839864-g75e9154c1: []string{"C:\\Program Files (x86)\\Tailscale IPN\\tailscaled.exe", "/subproc", "2646014839487dfcdbf94544078db3865f63fa7a9ce8cacd2b799d3b7c4c5570"}
2022-05-26 20:55:25.6205173 -0700 PDT: subproc mode: logid=2646014839487dfcdbf94544078db3865f63fa7a9ce8cacd2b799d3b7c4c5570
2022-05-26 20:55:25.7072178 -0700 PDT: tailscaled: getting engine... (try 1)
2022-05-26 20:55:25.8103194 -0700 PDT: ipnserver: rename .\..tmp1926937885 : The system cannot find the path specified.
2022-05-26 20:55:25.8293041 -0700 PDT: subprocess exited: exit status 1
2022-05-26 20:55:25.8293764 -0700 PDT: BabysitProc: backoff: 25868 msec
2022-05-26 20:55:51.7018772 -0700 PDT: exec: "C:\\Program Files (x86)\\Tailscale IPN\\tailscaled.exe" [/subproc 2646014839487dfcdbf94544078db3865f63fa7a9ce8cacd2b799d3b7c4c5570]
2022-05-26 20:55:52.1952422 -0700 PDT: Program starting: v1.25.91-tfc5839864-g75e9154c1: []string{"C:\\Program Files (x86)\\Tailscale IPN\\tailscaled.exe", "/subproc", "2646014839487dfcdbf94544078db3865f63fa7a9ce8cacd2b799d3b7c4c5570"}
2022-05-26 20:55:52.195808 -0700 PDT: subproc mode: logid=2646014839487dfcdbf94544078db3865f63fa7a9ce8cacd2b799d3b7c4c5570
2022-05-26 20:55:52.2356523 -0700 PDT: tailscaled: getting engine... (try 1)
2022-05-26 20:55:52.2856649 -0700 PDT: ipnserver: rename .\..tmp2651800448 : The system cannot find the path specified.
2022-05-26 20:55:52.3081378 -0700 PDT: subprocess exited: exit status 1
2022-05-26 20:55:52.3081378 -0700 PDT: BabysitProc: backoff: 37933 msec
```

### OS

Windows

### OS version

10

### Tailscale version

1.25.91

### Bug report

2646014839487dfcdbf94544078db3865f63fa7a9ce8cacd2b799d3b7c4c5570

**Comment by bradfitz** [comment]
I think @raggi just fixed this in https://github.com/tailscale/tailscale/commit/3f686688a6cff4c101c85487673d7181805829d1 perhaps? I haven't read/understood that change yet.

**Comment by DentonGentry** [comment]
Fixed in 1.25.98.
